app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -75,9 +83,9 @@
 
         <Button
             android:id="@+id/butLogin"
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
The error message is now no longer displayed because the login button takes the whole width. So either this needs to be reverted or the error message needs to be moved out of this LinearLayout. I think reverting might be good because then then button does not change its size when pressing it.